i2c/bmp280_i2c/bmp280_i2c.c

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
```

Comment:
I'm assuming you're employed by Trading and have done this example as part of that work: otherwise this looks a bit odd.